politeiad/backend/tlogbe/testing.go

```diff
@@ -163,6 +241,49 @@ func newTestTlogBackend(t *testing.T) (*tlogBackend, func()) {
 	}
 }
 
+func newTestCommentsPlugin(t *testing.T) (*commentsPlugin, *tlogBackend, func()) {
```

Comment:
Why do you need test plugins?  Why can't you use the regular plugin initialization functions?